.ci/openjdk14-excluded.files

```diff
@@ -495,3 +495,69 @@
 <module name="BeforeExecutionExclusionFileFilter">
   <property name="fileNamePattern" value="[\\/]CheckErrorsForSource7.java$"/>
 </module>
+<module name="BeforeExecutionExclusionFileFilter">
+  <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]tools[\\/]javac[\\/]T8185983[\\/]RejectTypeArgsOnSelectTest.java$"/>
```

Comment:
Full paths now, in response to comments at https://github.com/checkstyle/checkstyle/pull/8920#discussion_r516240425